boards/arm/nrf9160_innblue11/doc/index.rst

```diff
@@ -0,0 +1,203 @@
+.. _nrf9160_innblue11:
+
+nRF9160-INNBLUE11
+################
+
+Overview
+********
+
+The nRF9160 innblue v11 is a single-board development kit for evaluation
+and development on the nRF9160 SiP for LTE-M and NB-IoT. The
+nrf9160_innblue11 board provides support for the Nordic Semiconductor
+nRF9160 ARM Cortex-M33F CPU with ARMv8-M Security Extension and the
+following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+.. figure:: img/nrf9160_pca10090.png
+     :width: 711px
+     :align: center
+     :alt: nRF9160 PCA10090 DK
+
+     nRF9160 PCA10090 DK (Credit: Nordic Semi)
+
+More information about the board can be found at the
+`nRF91 DK website`_. The `Nordic Semiconductor Documentation library`_
+contains the processor's information and the datasheet.
+
+Hardware
+********
+
+nRF9160 DK has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 32 MHz.
+
+Supported Features
+==================
+
+The nrf9160_pca10090 board configuration supports the following

```

Comment:
You've switched board names here. Is this the right information about the nRF9160 innblue v11 board?